src/credentials/awsCredentialsStatusBarItem.ts

```diff
@@ -41,19 +42,18 @@ function handleDevSettings(devSettings: DevSettings, statusBarItem: vscode.Statu
     const developerMode = Object.keys(devSettings.activeSettings)
 
     if (developerMode.length > 0) {
-        ;(statusBarItem as any).backgroundColor = new vscode.ThemeColor('statusBarItem.errorBackground')
+        ;(statusBarItem as any).backgroundColor ??= new vscode.ThemeColor('statusBarItem.warningBackground')
```

Comment:
It's an ugly yellow now :/
I tried `statusBar.debuggingBackground` but it didn't work